sdk/endpoints/endpoints_config.go

```diff
@@ -2577,7 +2576,84 @@ const endpointsJson =`{
 			"code": "sts",
 			"document_id": "28756",
 			"location_service_code": "sts",
-			"regional_endpoints": null,
+			"regional_endpoints": [
+				{
+					"region": "eu-central-1",
+					"endpoint": "sts.eu-central-1.aliyuncs.com"
+				},
+				{
+					"region": "eu-west-1",
+					"endpoint": "sts.eu-west-1.aliyuncs.com"
+				},
+				{
+					"region": "me-east-1",
+					"endpoint": "sts.me-east-1.aliyuncs.com"
+				},
+				{
+					"region": "us-east-1",
+					"endpoint": "sts.us-east-1.aliyuncs.com"
+				},
+				{
+					"region": "us-west-1",
+					"endpoint": "sts.us-west-1.aliyuncs.com"
+				},
+				{
+					"region": "ap-south-1",
+					"endpoint": "sts.ap-south-1.aliyuncs.com"
+				},
+				{
+					"region": "ap-northeast-1",
+					"endpoint": "sts.ap-northeast-1.aliyuncs.com"
+				},
+				{
+					"region": "ap-southeast-5",
+					"endpoint": "sts.ap-southeast-5.aliyuncs.com"
+				},
+				{
+					"region": "ap-southeast-3",
+					"endpoint": "sts.ap-southeast-3.aliyuncs.com"
+				},
+				{
+					"region": "ap-southeast-2",
+					"endpoint": "sts.ap-southeast-2.aliyuncs.com"
+				},
+				{
+					"region": "ap-southeast-1",
+					"endpoint": "sts.ap-southeast-1.aliyuncs.com"
+				},
+				{
+					"region": "cn-hongkong",
+					"endpoint": "sts.cn-hongkong.aliyuncs.com"
+				},
+				{
+					"region": "cn-shanghai",
+					"endpoint": "sts.cn-shanghai.aliyuncs.com"
+				},
+				{
+					"region": "cn-hangzhou",
+					"endpoint": "sts.cn-hangzhou.aliyuncs.com"
+				},
+				{
+					"region": "cn-huhehaote",
+					"endpoint": "sts.cn-huhehaote.aliyuncs.com"
+				},
+				{
+					"region": "cn-zhangjiakou",
+					"endpoint": "sts.cn-zhangjiakou.aliyuncs.com"
+				},
+				{
+					"region": "cn-beijing",
+					"endpoint": "sts.cn-beijing.aliyuncs.com"
+				},
+				{
+					"region": "cn-qingdao",
+					"endpoint": "sts.cn-qingdao.aliyuncs.com"
+				},
+				{
```

Comment:
Hi  the region list is incomplete.

Reply:
I will trigger STS release a new version SDK to enable endpoint rule.

Reply:
I've put all regions according to https://www.alibabacloud.com/help/doc-detail/66053.htm

Reply:
@JacksonTian so when will this be merged or updated? No updates so far?

Reply:
You can pull the latest sts sdk. The problem has been solved.